articles/references/domain-designer/associations/associations.md

```diff
@@ -3,36 +3,34 @@ uid: references.domain-designer.associations
 ---
 # Domain Designer Associations
 
-Intent Architect's Domain Designer was originally built with the idea of being able to model Class Diagrams similar to how the UML specification would have it.
-Once a `Class` has been defined by its `members` and `operations` it would be useful to also model how it would relate with another class using some visual arrow and some symbol notation denoting the kind of relationship between the two `Classes`.
-Wikipedia provides some information regarding the concept of [Relationships](https://en.wikipedia.org/wiki/Class_diagram#Relationships) if you want to find out more.
+Intent Architect's Domain Designer enables modelling of a sub-set of [UML](https://en.wikipedia.org/wiki/Unified_Modeling_Language), in particular, [class diagrams](https://en.wikipedia.org/wiki/Class_diagram). It supports modelling of Classes, their Attributes, Operations (method invocations) and [Associations (relationships)](https://en.wikipedia.org/wiki/Class_diagram#Relationships) between classes. Additionally, [Stereotypes](https://en.wikipedia.org/wiki/Stereotype_(UML)) can be used to extend the vocabulary of elements on the diagram.
 
-In the UML world, an `Association` is a kind of `Relationship` while in Intent Architect's Domain Designer the term `Association` represents the kind of relationship between two `Entities` (or `Classes`). 
+> [!NOTE]
+> For more information on using Stereotypes in Intent Architect you can refer to [this article](xref:how-to-guides.use-stereotypes).
 
-The following information around associations can be modeled in Intent Architect:
+This article will focus on the different options available within the Domain Designer for modelling Associations and how to do so.
 
-- [Association Types](xref:references.domain-designer.associations#association-types) - Relationship ownership between Entities
-- [Multiplicity Types](xref:references.domain-designer.associations#multiplicity-types) - Cardinality between related Entities
-- [Navigability](xref:references.domain-designer.associations#navigability) - Constraining the direction of accessing Entities from other Entities
```

Comment:
DocFX already always shows a table of contents so I felt no need to also have it repeated here. I ensured each section had a description.

Also, just a quick tip, instead of:

`[Association Types](xref:references.domain-designer.associations#association-types)`

You can just do:

`[Association Types](#association-types)`